package.json

```diff
@@ -104,7 +104,7 @@
     "pegjs-loader": "^0.5.6",
     "platformicons": "^4.3.0",
     "po-catalog-loader": "2.0.0",
-    "prettier": "^2.6.0",
+    "prettier": "2.6.0",
```

Comment:
Had to change this to get the tooling to match correctly. Probably should be checking the lock file